backend/src/routes/group.ts

```diff
@@ -130,7 +118,7 @@ router.delete('/follower/:grouphandle/:userhandle', async (ctx, next) => {
 		return;
 	}
 	const user = await User.findOneBy({
-		handle: ctx.params.userhandle,
+		username: ctx.params.userhandle,
```

Comment:
here too, gotta use ctx.state.user.id then grab username from the user obj instead